config/.env.sample

```diff
@@ -2,8 +2,11 @@
 
 # Port that the server should listen on within the container (Optional)
 # PORT=4000
-# The server's domain, used for session configuration and OAuth redirects (use *.ngrok.io for local development)
+
+# This app server's hostname. For session configuration and OAuth redirects
+# Note: Use xxxxxxxxxxxx.ngrok.io for local development.
 DOMAIN=
+
```

Comment:
```suggestion
```
For consistency, I'd like to group the server stuff without newlines between the items. The groupings echo the nested objects from `config.ts`.

Reply:
OK.  I won't debate.  I think the blank line helps delineate the items so that it's not a dense block of text.  Esp. if the blank line before this item is removed, to a cursory glance will look like five lines of comments.